dogstatsd.py

```diff
@@ -440,7 +440,8 @@ def run(self):
             try:
                 self.server.start()
             except Exception as e:
-                log.exception('Error starting server')
+                log.exception(
+                    'Error starting dogstatsd server on {0}'.format(self.server.sockaddr))
```

Comment:
Ultra nitpick 👮 : it's usually preferable to let the `logging` module do the formatting, i.e.
```python
log.exception('Error starting dogstatsd server on %s', self.server.sockaddr)
```

It avoids to perform unnecessary or prematurely formatting (although in your case you're logging with the `EXCEPTION` level so it makes no difference).